src/utils/persistAccount.ts

```diff
@@ -0,0 +1,9 @@
+import createPersistedState from 'use-persisted-state';
+
+interface WalletAccount {
+    address: string;
```

Comment:
My only comment would be to use a `connected` member on this interface rather than relying on address string.
That's nit pick though and can easily be fixed through refactoring.